keen/api.py

```diff
@@ -190,6 +211,45 @@ def delete_events(self, event_collection, params):
         self.error_handling(response)
         return True
 
+    def get_collection(self, collection_name):
+        """
+        Extracts info about a collection using the Keen IO API. A master key must be set first.
+
+        :param event_name: the name of the collection to retrieve info for
+        """
+        if not self.master_key:
+            raise exceptions.InvalidEnvironmentError(
+                "The Keen IO API requires a master key to get events. "
+                "Please set a 'master_key' when initializing the "
+                "KeenApi object."
+            )
+        url = "{0}/{1}/projects/{2}/events/{3}".format(self.base_url, self.api_version,
+                                                       self.project_id, collection_name)
+        headers = {"Authorization": self.master_key}
+        response = self.fulfill(HTTPMethods.GET, url, headers=headers, timeout=self.get_timeout)
+        self.error_handling(response)
+
+        return response.json()
+
+    def get_all_collections(self):
+        """
+        Extracts info about all collections using the Keen IO API. A master key must be set first.
```

Comment:
How about "Return schema information for all the event collections in a given project.." (I'm using this [page](https://keen.io/docs/api/?shell#inspect-all-collections) in Keen's Docs for reference.)